### PR TITLE
Python 3.9 upgrade

### PR DESCRIPTION
**Pull request by webbyfox**
To do (delete all that do not apply):

 - [ ] Change has a jira ticket that has the correct status.
 - [ ] Changelog entry added.
 - [ ] (if there are vulnerable requirements) Upgraded any vulnerable dependencies.
 - [ ] (if updating requirements) Requirements have been compiled.
 - [ ] (if adding CSS) CSS have been compiled.
 - [ ] (if adding HTML) Change is accessible to the standards we subscribe to.
 - [ ] (if adding env vars) Added any new environment variable to vault.
 - [ ] (if adding feature flags) Cleaned up old flags
 - [ ] (if changing the UI) Add a printscreen to the PR
 - [ ] (if hotfix) Has made PR into develop too

### CODE REVIEW

File: manifest.yml [hunk 1]
```diff
@@ -1,4 +1,5 @@
 ---
 applications:
-  - buildpacks: 
-      - https://github.com/cloudfoundry/python-buildpack.git#v1.6.25
+  - buildpacks:
+      - python_buildpack
+    timeout: 180
```
Comment (stevejalim) [inline manifest.yml:5]: Is that timeout an addition or just that we're losing the empty last line in the file?

File: requirements.in [hunk 2]
```diff
@@ -18,3 +18,4 @@ django-redis==4.10.0
 django-storages==1.7.1
 boto3==1.7.30
 sentry-sdk==0.13.4
+directory-healthcheck
```
Comment (stevejalim) [inline requirements.in:21]: Same here - presumably just a loss of empty last line?

File: manifest.yml (hunk 1 above)
Comment (webbyfox) [reply manifest.yml:5]: It's an addition aligning with other repos. original picked from great-cms

**Review by stevejalim — APPROVED** [review]
LGTM as long as my assumptions about end-of-file blank lines is correct

**Pull request merged**